.github/workflows/ci.yml

```diff
@@ -77,6 +77,14 @@ jobs:
             pip install check-manifest
             ./run-tests.sh --check-manifest
 
+    lint-dockerfile:
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v2
+
+        - name: Lint Dockerfile
```

Comment:
```suggestion
        - name: Run hadolint
```

Reply:
I have used the REANA term, but indeed here we use more verbose "Check compliance..." so I have changed this to "Check Dockerfile compliance.".